semantic_conventions/resource/engine.yaml

```
@@ -0,0 +1,22 @@
+groups:
+  - id: engine_resource
+    prefix: engine
+    brief: >
+       Resource describing the packaged software running the application code. Engines are executed using process.runtime.
+    attributes:
+      - id: name
+        type: string
+        required: always
+        brief: >
+          The name of the engine.
+        examples: ['WildFly']
+      - id: version
+        type: string
+        brief: >
+          The version of the engine, as returned by the engine (preferably during runtime) without modification.
```

```suggestion
          The version of the engine.
```

I think the remainder is quite vague.